lxd/main_forkproxy.go

```diff
@@ -1023,6 +1023,10 @@ func parseAddr(addr string) (*proxyAddress, error) {
 	// Split into <protocol> and <address>
 	fields := strings.SplitN(addr, ":", 2)
 
+	if !shared.StringInSlice(fields[0], []string{"tcp", "udp", "unix"}) {
+		return nil, fmt.Errorf("Unknown connection type '%s'", fields[0])
+	}
```

Comment:
Not unreasonable to check here but we should make sure we catch this during config validation too.

Reply:
This check is still missing from the config validation code.

Reply:
This is done implicitly since the validation code runs `parseAddr`. I don't think we need to add this check again. What do you think?